### PR TITLE
45th Page Outline

### PR DESCRIPTION
**Pull request by Dr-N0**
This isn't finished yet, waiting on more information about 45th when we get to it. This is just the outline + the links and all that good stuff + funny 404 message

### CODE REVIEW

File: 45th/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: 45th Aniversary
```
Comment (RamZallan) [inline 45th/index.html:3]: *`Anniversary`, missing an `n`

File: _includes/nav.html [hunk 2]
```diff
@@ -29,6 +29,7 @@
                 <li class="nav-item"><a {% if page.active == 'blog' %}class="active"{% endif %} href="/blog/">Blog</a></li>
                 <li class="nav-item"><a target="_blank" rel="noopener" href="https://www.rit.edu/giving/ComputerScienceHouse">Donate</a></li>
                 <li class="nav-item"><a {% if page.active == 'contact' %}class="active"{% endif %} href="/contact/">Contact</a></li>
+                <li class="nav-item"><a {% if page.active == '45th' %}class="active"{% endif %} href="/45th/">45th Aniversary</a></li>
```
Comment (RamZallan) [inline _includes/nav.html:32]: Same typo as above, *Anniversary

File: 45th/index.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+active: 45th
+---
+
+<div class="gray-wrapper">
+    <div class="container spaced">
+        <div class="row">
+            <div class="col-12 col-lg-6">
+                <h1 class="header">45th Aniversary Celebration!</h1>
```
Comment (RamZallan) [inline 45th/index.html:11]: *Anniversary

File: 404.html [hunk 4]
```diff
@@ -8,6 +8,7 @@
             <div class="col">
                 <h2>Page not found :(</h2>
                 <p>The requested page could not be found.</p>
+                <p>(Press F to pay respects)</p>
```
Comment (stevenmirabito) [inline 404.html:11]: 👏 Bring 👏 back 👏 the 👏 video

**Comment by Dr-N0** [comment]
@RamZallan That should be good, lmk if there are any other issues

**Comment by mxmeinhold** [comment]
Looks like you're still including commits that are already merged. I can give more context when I get back to my computer.

**Comment by mxmeinhold** [comment]
I believe there are only 2 commits that are relevant to this PR, 7b643f7 and 4eadf2e. To clean this up, I recommend resetting to `ComputerScienceHouse/CSHPublicSite develop` and running `git cherry-pick 7b643f7 4eadf2e` and then force pushing to your branch.

**Comment by Dr-N0** [comment]
Going to remake the PR with propper changes, usually I wouldn't but since this change is small I don't mind

**Pull request closed without merging**